### ISSUE TITLE
Missing styles

### ISSUE THREAD

**Issue by matvp91** [scalar/scalar#3247]
### What happens?

I ran the HTML sample, and noticed that some styles are off:

<img width="307" alt="image" src="https://github.com/user-attachments/assets/d5bac1f4-7d2d-4e70-a08b-386df06a38b7">

<img width="439" alt="image" src="https://github.com/user-attachments/assets/4d2d2225-d2b5-490b-985b-bdf462a61afc">

```html
<!doctype html>
<html>
  <head>
    <title>Scalar API Reference</title>
    <meta charset="utf-8" />
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1" />
  </head>
  <body>
    <!-- Need a Custom Header? Check out this example https://codepen.io/scalarorg/pen/VwOXqam -->
    <script
      id="api-reference"
      data-url="https://cdn.jsdelivr.net/npm/@scalar/galaxy/dist/latest.yaml"></script>
    <script src="https://cdn.jsdelivr.net/npm/@scalar/api-reference"></script>
  </body>
</html>
```

### What did you expect to happen?

To have proper styling.

**Comment by Nikkoro** [comment]
Duplicate, already mentioned at #3158. Bug occuring on Chromium based browsers

**Comment by NotReallyPrince** [comment]
![Screenshot_2024-09-25-21-13-07-75_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/faea910f-426d-4d75-94d1-f01c63c021e9)


Bug in chromium based browser

**Comment by gsmcdonald** [comment]
It looks like the mobile header is being shown incorrectly on desktops (and the sidebar incorrectly being shown on mobile) with the latest versions of Chromium. We're seeing similar results.

**Comment by hwkr** [comment]
Should be fixed by #3253!
